docs/migration/css-changes-in-version-4.md

```diff
@@ -143,6 +143,14 @@ title: Changes to Styles in 4.0
 ## Changes in Card Component 
```

Comment:
Please rename to ## Changes in Product Configurator Card Component, and have this section together with other configurator related sections

Reply:
Done